### PR TITLE
[Edit]Semantic HTML: Content Consolidation

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

There are two entries within docs covering semantic HTML. This edit removes the semantic HTML entry from the general topics directory and consolidates the content from that entry into the entry under the HTML directory.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by yangc95 — APPROVED** [review]

File: content/html/concepts/semantic-html/semantic-html.md [hunk 1]
```diff
@@ -33,8 +39,8 @@ With Semantic HTML, we were introduced to elements that tell developers and brow
 
 ## Why Use Semantic HTML?
 
-- Accessibility: Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are able to interpret the code better.
+- Accessibility: Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are better able to interpret the content of the site.
 
 - Readability: Semantic HTML also makes the website’s source code easier to read for other web developers.
 
-- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit your webpage. With better SEO, search engines are better able to identify the content of your website and weight the most important content appropriately.
+- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit your webpage. With better SEO, search engines are better able to identify the content of your website and weigh the most important content appropriately.
```
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/semantic-html.md:46]:
```suggestion
- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit a webpage. With better SEO, search engines are better able to identify the content of a website and weigh the most important content appropriately.
```
Getting rid of the second-person pronoun.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@caupolicandiaz one comment.

File: content/html/concepts/semantic-html/semantic-html.md (hunk 1 above)
Comment (caupolicandiaz) [reply content/html/concepts/semantic-html/semantic-html.md]: @SSwiniarski All set. Thanks

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz looks good.

**Pull request merged**